cache-caffeine/src/main/java/io/micronaut/cache/caffeine/DefaultSyncCache.java

```diff
@@ -81,7 +79,7 @@ public DefaultSyncCache(
      */
     @Inject
     public DefaultSyncCache(
-            CacheConfiguration cacheConfiguration,
+            CaffeineCacheConfiguration cacheConfiguration,
```

Comment:
This breaks binary compatibility, you would need to revert this change and introduce an instanceof check

Reply:
Done